schema/data/deploy-data.sh

```diff
@@ -192,6 +192,7 @@ deployProdData() {
   _psql -f "./prod/emission.sql"
   _psql -f "./prod/gas.sql"
   _psql -f "./prod/emission_gas.sql"
+  _psql -f "./prod/application_validation_function.sql"
```

Comment:
This needs to be added to `deployTestData` below, as that is the one that is actually run by cypress